mirage/serializers/registration-provider.ts

```diff
@@ -24,6 +24,14 @@ export default class RegistrationProviderSerializer extends ApplicationSerialize
                     },
                 },
             },
+            brand: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/providers/registrations/${model.id}/brand/`,
```

Comment:
this should point to brand detail route -- i think this'll break if we try to access `provider.brand` without having already fetched it via embedding
```suggestion
                        href: `${apiUrl}/v2/brands/${model.brand.id}/`,
```